opnode/bss/config.go

```diff
@@ -0,0 +1,224 @@
+package bss
+
+import (
+	"errors"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/urfave/cli"
+
+	"github.com/ethereum-optimism/optimistic-specs/opnode/bss/flags"
+)
+
+var (
+	// ErrSequencerPrivKeyOrMnemonic signals that the user tried to set both
+	// sequencer wallet derivation methods or neither of them.
+	ErrSequencerPrivKeyOrMnemonic = errors.New("either sequencer-private-key " +
```

Comment:
Splitting the line helps with line length, but makes it hard to grep for the exact error from the error string.

Reply:
This error has been removed, but i did a quick grep and it seems like all of the other instances are on one line 👍